### ISSUE TITLE
Why is this a recommendation for every object I make?

### ISSUE THREAD

**Issue by EzequielDM** [codota/TabNine#459]
**please complete the following information:**	
 - OS version: Windows 10	
 - Editor: vscode	
 - Editor version: 1.60.2
 - Programming language: ts	
 - TabNine extension version: 3.4.27
 - Engine version (could be located within Tabnine Hub): 3.6.8-40

**Issue Details:**
Almost every time I open curly brackets, it suggests "teardown" and I have no idea what that's supposed to be and why, no matter the context, it shows "teardown" as a suggestion.
On the other files after writing some code it stopped showing "teardown" as an option, but on new files, it still likes "teardown" a lot.

![Example](https://i.imgur.com/kOpnFKo.png)

gz#6870

**Comment by EzequielDM** [comment]
Also, complementing with more examples.

![Example 2](https://i.imgur.com/f1Bn04g.png)

It doesn't seem to care where it's or what context, if it's the second object, it shows "teardown"

![Example 3](https://i.imgur.com/v8k5wqM.png)

**Comment by maya-tabnine** [comment]
To figure out the cause, you need to locate your binary files and run the engine directly.

On Tabnine Hub, scroll down to the Engine file path, and find the Tabnine file path.
Try to run the following command from the terminal using the right path:

`PATH_TO_ENGINE/TabNine.cmd --log_to_stderr`

Also, there are logs at:
` C:\Users\{UserName}\AppData\Code\User\globalStorage\tabnine.tabnine-vscode\binaries\{Binary_Version}\x86_64-pc-windows-gnu/`

Would you please collect the output to a file and share it with us?

**Comment by jaques-sam** [comment]
I have the same:

![image](https://user-images.githubusercontent.com/42445148/146172401-5eac6063-f73b-48b1-826d-a753dc785be2.png)

Unfortunately I've the free version where this cannot be changed in the settings.

@maya-tabnine I've tried your recommendation, but it doesn't use that engine in VSCode... I tried again by disabling the Tabnine extension but that doesn't show me (suprise suprise) any tabnine autocomplete. In both cases the logs are not changed when doing that. How do I run the engine in VSCode with the extra parameter? Thx!
